### PR TITLE
Update README.md with instructions for setting up

### PR DESCRIPTION
**Pull request by kirkins**
Hello, my friend is new to programming and he wanted to use this library as an experiment.

He had trouble finding the inline instructions for settings up the `args.yaml` file. I've added some instructions in the README and a template based on your code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,16 @@ gutengrep.py "^\W*([a-zA-ZåäöÅÄÖàéÉÈíšŠ]+['[a-zA-Z]+]?[- :;,.¿?\!]
...
+Before you can use  this script you will need to get API credentials from Twitter.
+
+Go to https://developer.twitter.com/en/apps and create a new app. Once approved you will be able to access the credentials mentioned below.
+
+    consumer_key: TODO_ENTER_YOURS
+    consumer_secret: TODO_ENTER_YOURS
+    access_token: TODO_ENTER_YOURS
+    access_token_secret: TODO_ENTER_YOURS
+
+Open the `args.yaml` file and replace each instance of `TODO_ENTER_YOURS` with the value from your twitter app.
```
Comment (hugovk) [inline README.md]:
```suggestion
Open the `args.yaml` file and replace each instance of `TODO_ENTER_YOURS` with the value from your Twitter app.
```

File: README.md [hunk 2]
```diff
@@ -17,3 +17,16 @@ gutengrep.py "^\W*([a-zA-ZåäöÅÄÖàéÉÈíšŠ]+['[a-zA-Z]+]?[- :;,.¿?\!]
 # Remove blanks and duplicates:
 sort /tmp/1.txt | uniq > six-word-sentences.txt  # 
 ```
+
+## Setup
+
+Before you can use  this script you will need to get API credentials from Twitter.
+
+Go to https://developer.twitter.com/en/apps and create a new app. Once approved you will be able to access the credentials mentioned below.
```
Comment (hugovk) [inline README.md]:
```suggestion
Go to https://developer.twitter.com/en/apps and create a new app. Once approved you will be able to access the credentials below.
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Good idea, thanks for the PR!

Let's also rename `args.yaml` to `randomsentencebot_example.yaml`, that's how I've done it for some other bots:

* https://github.com/hugovk//everyfinnishno
* https://github.com/hugovk/mass_shoot_bot

File: randomsentencebot.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def get_twitter():
     global TWITTER
 
     if TWITTER is None:
-        data = load_yaml(args.yaml)
+        data = load_yaml(randomsentencebot_example.yaml)
```
Comment (hugovk) [inline randomsentencebot.py]: Let's not change this line: it's a variable, not a file name (which would be a string). We don't need to edit the `.py` file for this PR.

```suggestion
        data = load_yaml(args.yaml)
```

**Comment by kirkins** [comment]
@hugovk I updated references in the code to the new name. Or were you expecting the user to rename the file?

Perhaps `config.yaml` would be a good name as this is where people expect to make changes to to a project for setup.

**Comment by kirkins** [comment]
Ok I see what you mean.

**Comment by hugovk** [comment]
```console
$ python randomsentencebot.py --help
usage: randomsentencebot.py [-h] [-y YAML] [-i INFILE] [--hashtags HASHTAGS]
                            [-nw] [-x] [-q]

Tweet a random line from a text file.

optional arguments:
  -h, --help            show this help message and exit
  -y YAML, --yaml YAML  YAML file location containing Twitter keys and secrets
                        (default:
                        /Users/hugo/Dropbox/bin/data/randomsentencebot.yaml)
  -i INFILE, --infile INFILE
                        A random line is chosen from this text file (default:
                        /Users/hugo/Dropbox/bots/six-worders12.txt)
  --hashtags HASHTAGS   Comma-separated list of random hashtags (default: #Six
                        WordStories,#SixWordStory,#6WordStory,#6WordStories,No
                        ne)
  -nw, --no-web         Don't open a web browser to show the tweeted tweet
                        (default: False)
  -x, --test            Test mode: go through the motions but don't update
                        anything (default: False)
  -q, --quiet           Only print out tweet (and errors) (default: False)
```

I use `randomsentencebot.yaml` as my file, and the user will need to rename `randomsentencebot_example.yaml` to that if they want, and then it as their input to the `--yaml` argument.

(I see I've put a full path as the default for `--yaml`, but some bots don't have that and the real data is in the same directory as the Python script. So it's called `*_example.yaml` to make sure the secrets aren't committed to Git.)

File: README.md [hunk 4]
```diff
@@ -17,3 +17,16 @@ gutengrep.py "^\W*([a-zA-ZåäöÅÄÖàéÉÈíšŠ]+['[a-zA-Z]+]?[- :;,.¿?\!]
...
+Before you can use  this script you will need to get API credentials from Twitter.
+
+Go to https://developer.twitter.com/en/apps and create a new app. Once approved you will be able to access the credentials below.
+
+    consumer_key: TODO_ENTER_YOURS
+    consumer_secret: TODO_ENTER_YOURS
+    access_token: TODO_ENTER_YOURS
+    access_token_secret: TODO_ENTER_YOURS
+
+Open the `args.yaml` file and replace each instance of `TODO_ENTER_YOURS` with the value from your Twitter app.
```
Comment (hugovk) [inline README.md]: Still an `args.yaml` here, let's update this bit too.

File: README.md [hunk 5]
```diff
@@ -29,4 +29,4 @@ Go to https://developer.twitter.com/en/apps and create a new app. Once approved
     access_token: TODO_ENTER_YOURS
     access_token_secret: TODO_ENTER_YOURS
 
-Open the `args.yaml` file and replace each instance of `TODO_ENTER_YOURS` with the value from your Twitter app.
+Open the `randomsentencebot.py` file and replace each instance of `TODO_ENTER_YOURS` with the value from your Twitter app.
```
Comment (hugovk) [inline README.md]: Not the Python file :)

**Comment by hugovk** [comment]
I made a little edit. Thanks!

**Pull request merged**